controllers/articlesEndPointController.js

```diff
@@ -0,0 +1,22 @@
+const { getArticleById } = require("../models/articlesEndPointModel");
+
+const getSinlgeArticle = (req, res, next) => {
+  const articleID = req.params.article_id;
+
+  if (isNaN(articleID)) {
+    return res.status(400).send({ msg: "Bad Request" });
+  }
```

Comment:
You don't have to have this logic here for `isNaN(articleID)` - you can let PSQL throw this error for you, if the ID is a string it will give you an error you can handle